src/test/components/TimelineMarkers.test.js

```diff
@@ -41,7 +41,6 @@ describe('TimelineMarkers', function() {
       .mockImplementation(() => ctx);
 
     const profile = getProfileWithMarkers([
-      ['GCMajor', 2, { startTime: 2, endTime: 12 }],
```

Comment:
Any reason you decided to remove it?
If you keep it this would actually test some things: filtering in the main marker track, and test the memory marker track.

Reply:
Because the payload wasn't correct for a GCMajor. Mainly it's cumbersome to create these payloads due to their complexity. I could create a utility function to create one and put it back if you feel that it's valuable.

Reply:
ah yeah I see.
For your code to work, I guess it mainly needs the `type`, right?